spec/export/index.md

```diff
@@ -135,18 +135,72 @@ Export data from a FHIR server, whether or not it is associated with a patient.
       If the client explicitly asks for export of resources that the bulk data server doesn't support, the server SHOULD return details via an OperationOutcome resource in an error response to the request.<br /><br />
       For example <code>_type=Observation</code> could be used to filter a given export response to return only Observation resources.</td>
     </tr>
+    <tr>
+      <td><code>_elements</code></td>
+      <td><span class="label label-info">optional, experimental</span></td>
+      <td><span class="label label-info">optional</span></td>
+      <td>string of comma-delimited FHIR Elements</td>
+      <td>When provided, the server SHOULD omit unlisted, non-mandatory elements from the resources returned. Elements should be of the form <code>[resource type].[element name]</code> (eg. <code>Patient.id</code>) or <code>[element name]</code> (eg. <code>id</code>) and only root elements in a resource are permitted. If the resource type is omitted, the element should be returned for all resources in the response where it is applicable..<br /><br />
+        Servers are not obliged to return just the requested elements. Servers SHOULD always return mandatory elements whether they are requested or not. Servers SHOULD mark the resources with the tag SUBSETTED to ensure that the incomplete resource is not actually used to overwrite a complete resource.<br/><br/>
```

Comment:
should it also list the recommendations from the base spec on clients?
> Clients SHOULD list all mandatory and modifier elements in a resource as part of the list of elements.

Reply:
I'm probably being obtuse here, but I don't see what that brings to the spec. On the server side, the mandatory elements text supports the behavior of servers returning more data than was requested, but on the client side, if they request mandatory fields they'll get them and if they don't, they may or may not depending on the server behavior. Not sure why we think they "should" request them...

Reply:
Yeah, I don't disagree.  I guess its more just a recommendation to protect them from themselves by ensuring that they'll always get back a valid resource and that they aren't missing any elements that could modify the meaning of the subsetted resource they received.  Feel free to mark this one as resolved.